src/app/debug/filter-side-drawer/filter.service.ts

```diff
@@ -6,17 +6,18 @@ import { debounceTime, filter, Observable, Subject } from 'rxjs';
 })
 export class FilterService {
   private showFilterSubject: Subject<boolean> = new Subject();
-  private metadataLabelsSubject: Subject<string[]> = new Subject();
+  private metadataLabelsSubject: Subject<Array<string>> = new Subject();
   private filterContextSubject: Subject<Map<string, string>> = new Subject();
   private currentRecordsSubject: Subject<Map<string, Array<string>>> = new Subject();
   private metadataTypesSubject: Subject<Map<string, string>> = new Subject();
   private filterErrorSubject: Subject<[boolean, Map<string, string>]> = new Subject();
   private filters: Map<string, string> = new Map<string, string>();
   private metadataTypes: Map<string, string> = new Map<string, string>();
+  private metadataLabels: Array<string> = new Array<string>();
   private filterErrors: Map<string, string> = new Map<string, string>();
 
   showFilter$: Observable<boolean> = this.showFilterSubject.asObservable();
-  metadataLabels$: Observable<string[]> = this.metadataLabelsSubject.asObservable();
+  metadataLabels$: Observable<Array<string>> = this.metadataLabelsSubject.asObservable();
   currentRecords$: Observable<Map<string, Array<string>>> = this.currentRecordsSubject.asObservable();
   metadataTypes$: Observable<Map<string, string>> = this.metadataTypesSubject.asObservable();
   filterError$: Observable<[boolean, Map<string, string>]> = this.filterErrorSubject.asObservable();
```

Comment:
Please order these class variables to be in line with [our conventions](https://github.com/frankframework/frontend-conventions/blob/master/typescript.md#class-member-ordering)

Reply:
Added the change.